packages/ckeditor5-dev-env/lib/translations/collect-utils.js

```diff
@@ -0,0 +1,202 @@
+	 * @returns {Array<String>}
+	 */
+	getMissingContextErrorMessages( contexts, translations ) {
+		const errors = [];
+
+		for ( const translation of translations ) {
+			const errorMessage = utils.maybeGetContextErrorMessage( contexts, translation );
+
+			if ( errorMessage ) {
+				errors.push( errorMessage );
+			}
+		}
+
+		return errors;
+	},
+
+	maybeGetContextErrorMessage( contexts, translation ) {
+		const packageContext = contexts.get( translation.package );
+		const corePackageContext = contexts.get( corePackageName );
+		let error;
+
+		if ( !corePackageContext ) {
+			error = `${corePackageName}/lang/contexts.json file is missing.`;
+		}
+
+		else if ( !corePackageContext.content[ translation.key ] && !packageContext ) {
+			error = `contexts.json file or context for the translation key is missing (${ translation.package }, ${ translation.key }).`;
+		}
+
+		else if ( !corePackageContext.content[ translation.key ] && !packageContext.content[ translation.key ] ) {
+			error = `Context for the translation key is missing (${ translation.package }, ${ translation.key }).`;
+		}
+
+		return error;
+	},
+
+	getUnusedContextErrorMessages( contexts, translations ) {
+		const usedContextMap = new Map();
+
+		for ( const [ packageName, context ] of contexts ) {
+			for ( const key in context.content ) {
+				usedContextMap.set( packageName + '/' + key, false );
+			}
+		}
+
+		for ( const translation of translations ) {
+			usedContextMap.set( translation.package + '/' + translation.key, true );
+			usedContextMap.set( corePackageName + '/' + translation.key, true );
+		}
+
+		return [ ...usedContextMap ]
+			.filter( ( [ key, usage ] ) => !usage )
+			.map( ( [ key ] ) => `Unused context: ${ key }.` );
+	},
+
+	getRepeatedContextErrorMessages( contexts ) {
+		const errors = [];
+		const keys = new Set();
+
+		for ( const context of contexts.values() ) {
+			for ( const key in context.content ) {
+				if ( keys.has( key ) ) {
+					errors.push( `Context is duplicated for the key: ${ key }.` );
+				}
+				keys.add( key );
+			}
+		}
+
+		return errors;
+	},
+
+	createPotFileContent( context ) {
+		const translationObjects = Object.keys( context.content ).map( str => ( {
+			id: str,
+			str: str,
+			ctxt: context.content[ str ]
+		} ) );
+
+		return utils.stringifyTranslationObjects( translationObjects );
+	},
+
+	stringifyTranslationObjects( translationObjects ) {
+		return translationObjects.map( ( translationObject ) => {
+			// Note that order is important.
+			return [
+				`msgctxt "${ translationObject.ctxt }"`,
+				`msgid "${ translationObject.id }"`,
+				`msgstr "${ translationObject.str }"`
+			].map( x => x + '\n' ).join( '' );
+		} ).join( '\n' );
+	},
+
+	savePotFile( packageName, fileContent ) {
+		const outputFilePath = path.join( process.cwd(), 'build', '.transifex', packageName, 'en.pot' );
+
+		fs.outputFileSync( outputFilePath, fileContent );
+
+		logger.info( `Created file: ${ outputFilePath }` );
```

Comment:
Didn't I comment on this already? :D Maybe it was in @pomek's PR. If the console log says something different than the function name then you know that something's wrong. In this case it's the log message.